backstop/tests/tabs.html

```diff
@@ -696,11 +1242,11 @@ <h2>Borderless</h2>
       <br />
 
       <div
-        class="iui-tabs-wrapper"
+        class="iui-tabs-wrapper iui-vertical iui-not-animated"
```

Comment:
Huh?
![vertiacl-tabs-glitch](https://user-images.githubusercontent.com/36186912/129673256-28ed0394-400c-44a2-af35-2ba41abdf5be.gif)

Reply:
I guess the classname `iui-not-animated` isn't exactly true 🤔.  Maybe it should be named `.iui-less-animated` but that doesn't sound great.

Reply:
Bring back `iui-idk`